### ISSUE TITLE
XML output no longer contains non-ASCII characters

### ISSUE THREAD

**Issue by sbraz** [MediaArea/MediaInfoLib#986]
Hi,
As reported [here](https://github.com/sbraz/pymediainfo/issues/58), v18.08 ignores UTF-8 characters in XML output. The culprit is 1116a70613cba13322fd3bbef906e1ed569c3cf4.
This can easily be reproduced with `--output=XML` or `--output=OLDXML` and https://github.com/sbraz/pymediainfo/blob/v2.3.0/tests/data/sample.mkv:
```xml
<Title>Ds Nol o un zphyr ha me vt de glaons wrmiens je dne dexquis rtis de buf au kir  la dge mr &amp; ctera !</Title>                                                                                            
```
instead of
```xml
<Title>Dès Noël où un zéphyr haï me vêt de glaçons würmiens je dîne d’exquis rôtis de bœuf au kir à l’aÿ d’âge mûr &amp; cætera !</Title>
```

BTW, you have not tagged (or not pushed) the `v18.08` tag, can you do it?

**Comment by JeromeMartinez** [comment]
Confirmed, big issue.

**Comment by g-maxime** [comment]
Fixed by #989 in version 18.08.1
